src/Controls/PlanarControls.js

```diff
@@ -1099,6 +1116,10 @@ class PlanarControls extends THREE.EventDispatcher {
      * @ignore
      */
     onContextMenu(event) {
+        if (!this.enabled) {
+            return;
+        }
+
```

Comment:
Doing this will allow the right-click menu to pop when right-clicking on a PlanarView with disabled controls. Currently, it is prevented and I don't think this behavior should change, but I might miss some use case. Perhaps you know of such an use case @valentinMachado and @jailln ?